src/main/scala/ai/privado/languageEngine/java/processor/JavaProcessor.scala

```diff
@@ -89,7 +90,24 @@ object JavaProcessor {
           )
           println(s"${Calendar.getInstance().getTime} - Brewing result...")
           MetricHandler.setScanStatus(true)
-          // Exporting
+          // Exporting Results
+          if (ScanProcessor.config.testOutput) {
+            JSONExporter.IntermediateFileExport(
+              outputIntermediateFileName,
+              sourceRepoLocation,
+              DataFlowCache.getIntermediateDataFlow
+            ) match {
+              case Left(err) =>
+                MetricHandler.otherErrorsOrWarnings.addOne(err)
+                Left(err)
+              case Right(_) =>
+                println(
+                  s"${Calendar.getInstance().getTime} - Successfully exported intermediate output to '${AppCache.localScanPath}/$outputIntermediateFileName' folder..."
```

Comment:
the path is incorrect, it should also hold `outputDirectoryName` in it

Reply:
@ankit-privado can we get this fixed and close this one?

Reply:
okay sure

Reply:
Yes done, @khemrajrathore PTAL